packages/application-shell/src/apollo-links/token-retry-link.ts

```diff
@@ -18,34 +20,48 @@ type ApolloContext = {
 // so that the MC BE can issue a new token.
 // NOTE: the retry is not meant to work for the MC access token.
 
+const TOKEN_RETRY_HEADER_NAME = 'X-Force-Token';
+
 export const getDoesGraphQLTargetSupportTokenRetry = (
   requestHeaders: ApolloContext['headers']
-) => {
+): boolean => {
```

Comment:
Otherwise we have an ESLint warning. I like type inference but our linting should comply I think.